library/src/main/java/com/novoda/downloadmanager/lib/DownloadThread.java

```diff
@@ -673,6 +677,20 @@ private void handleEndOfStream(State state) throws StopRequestException {
         }
     }
 
+    private void pause() throws StopRequestException {
```

Comment:
pausing a download vs pausing a batch not sure if we need both

https://github.com/novoda/download-manager/blob/master/library/src/main/java/com/novoda/downloadmanager/lib/DownloadManager.java#L442-L446